README.md

```diff
@@ -1,65 +1,47 @@
+The image is Alpine based, and includes Docker and Docker Compose, as well as Bash.
 
-Here is an example of Concourse [job](http://concourse.ci/concepts.html) that uses ```amidos/dcind``` image to run a bunch of containers in a task, and then runs the integration test suite. You can find the full example in the ```example``` directory.
+This image is an updated version of [karlkfi/concourse-dcind](https://hub.docker.com/r/karlkfi/concourse-dcind/).
+
+Inspired by [meAmidos/dcind](https://github.com/meAmidos/dcind), [concourse/docker-image-resource](https://github.com/concourse/docker-image-resource/blob/master/assets/common.sh), and [mesosphere/mesos-slave-dind](https://github.com/mesosphere/mesos-slave-dind).
+
+## Example
+
+Here is an example of a Concourse [job](http://concourse.ci/concepts.html) that uses `buildo/dcind` image to run a bunch of containers in a task, and then runs the integration test suite.
 
 ```yaml
+jobs:
   - name: integration
     plan:
-      - aggregate:
-        - get: code
-          params: {depth: 1}
-          passed: [unit-tests]
-          trigger: true
-        - get: redis
-          params: {save: true}
-        - get: busybox
-          params: {save: true}
-      - task: Run integration tests
+      - get: code
+        params:
+          depth: 1
+        passed:
+          - unit-tests
+        trigger: true
+      - task: integration-tests
         privileged: true
         config:
           platform: linux
           image_resource:
             type: docker-image
             source:
-              repository: amidos/dcind
+              repository: buildo/dcind
           inputs:
             - name: code
-            - name: redis
-            - name: busybox
           run:
-            path: sh
+            path: entrypoint.sh
             args:
-              - -exc
+              - bash
+              - -ceux
               - |
-                source /docker-lib.sh
-                start_docker
-
-                # Strictly speaking, preloading of images is not required.
-                # However you might want to do it for a couple of reasons:
-                # - If the image is from a private repository, it is much easier to let concourse pull it,
-                #   and then pass it through to the task.
-                # - When the image is passed to the task, Concourse can often get the image from its cache.
-                docker load -i redis/image
-                docker tag "$(cat redis/image-id)" "$(cat redis/repository):$(cat redis/tag)"
-
-                docker load -i busybox/image
-                docker tag "$(cat busybox/image-id)" "$(cat busybox/repository):$(cat busybox/tag)"
-
-                # This is just to visually check in the log that images have been loaded successfully
-                docker images
-
```

Comment:
Personally I would keep this bit about image preloading as it provides an example for a useful pattern